pilz_robot_programming/src/pilz_robot_programming/commands.py

```diff
@@ -598,13 +598,14 @@ def __init__(self):
 
     def append(self, cmd, blend_radius=0):
         """Adds the given robot motion command to the sequence. Currently, blending is only supported
-        for :py:class:`Lin` commands, however other commands can be used in a sequence if the blend radius is 0.
```

Comment:
Currently gripper is not working. See https://github.com/PilzDE/pilz_industrial_motion/pull/63.